kalamine/layout.py

```diff
@@ -43,6 +43,17 @@ def load_descriptor(file_path: Path) -> Dict:
             ext = load_descriptor(parent_path)
             ext.update(cfg)
             cfg = ext
+        if "version" in cfg:
+            version_check = cfg["version"].split(".")
+            if len(version_check) > 3:
+                raise Exception(
+                    f"Layout version number **must** follow `x.y.z` format\nCurrently got `version={cfg['version']}`"
```

Comment:
```suggestion
                    f"""Layout version number **must** follow `x.y.z` format
                    Currently got `version={cfg['version']}`"""
```

Reply:
suggestion works fine as well if “Currently“ is at the beginning of the line and not indented.